data/json/items/gun/300.json

```diff
@@ -18,5 +18,54 @@
     "barrel_length" : 2,
     "magazines": [ [ "300", [ "m2010mag" ] ] ],
     "flags": "NEVER_JAMS"
+  },
+  {
+    "id": "weatherby_5",
+    "copy-from": "rifle_manual",
+    "type": "GUN",
+    "name": "Weatherby Mark V",
+    "name_plural": "Weatherby Mark V",
+    "description": "The Weatherby Mark V is one of the finest bolt action rifles.  It was introduced in 1957.  These rifles are considered prestigious or luxury firearms.",
+    "weight": 4200,
+    "volume": 12,
+    "price": 259500, "//": "MSRP, believe it or not.",
+    "bashing": 12,
+    "to_hit": -1,
+    "material": [ "steel", "wood" ],
+    "color": "brown",
+    "ammo": "300",
+    "sight_dispersion": 60,
+    "aim_speed" : 6,
```

Comment:
I am assuming the preexisting aim speed was intentional. If it was purely game balance, I will remove aim-speed